Library/Formula/postgresql.rb

```diff
@@ -77,8 +82,16 @@ def install
   end
 
   def post_install
-    unless File.exist? "#{var}/postgres"
-      system "#{bin}/initdb", "#{var}/postgres"
+    system "pkill postgres || true"
+
+    if (var/"postgres").exist?
+      mv var/"postgres", var/"postgres-#{version_suffix}"
```

Comment:
What's the thinking behind the symlinking, if we're trying to ditch the plain ` postgres ` {var} folder altogether in favour of versioned dirs? Apologies if I'm being dim.